### ISSUE TITLE
[bug] rename category not assigning games to renamed category

### ISSUE THREAD

**Issue by eclay42** [rallion/depressurizer#19]
The rename category command isn't reassigning the games to the new category. It appears to delete the old category and create a new one, but any games in the deleted category are not automatically assigned to the new one so the renamed category ends up being empty.

**Comment by rallion** [comment]
Could you try this again, and check something for me? After renaming the category and confirming that it's empty or missing items, find one of the games that should be assigned (under <All>) and see if it actually shows the category in its category list.

Thanks!

**Comment by eclay42** [comment]
I just tried that and the games that didn't make it didn't list themselves as being in the new category. Some additional information: the first category I just renamed which only had a few games appeared to rename everything correctly, another category with more games (10-20) only had one game correctly reassigned to a new category. A much larger category (100+ games) appeared to work correctly, but I didn't exhaustively check each game. Another small category with 2 games lost both of them.

The only possible correlation I see is that the categories with spaces in them appear to take longer to process and are more likely to lose games.
